gamification/process/update.py

```diff
@@ -135,6 +137,37 @@ def get_participant_objects():
     return participants
 
 
+def get_meta_review_participant_objects():
+    return meta_review_participants.objects.all()
+
+
+def update_participants_data_with_meta_review(meta_review_participant):
+    """
+    Update participants based on meta-review
+
+    This method updates every participant based on the meta-review
+    received or given. If the meta-review is completed then it calls
+    'Did a meta-review or received a meta-review' activity.
+    """
+    if meta_review_participant.login == active_newcomers:
+        if (meta_review_participant.pos_in > 0 or
+            meta_review_participant.pos_out > 0 or
+            meta_review_participant.neg_in > 0 or
+                meta_review_participant.neg_out > 0):
+            """
+            Add a meta-review activity to the participant
+            """
+            name = meta_review_participant
+            activity = 'Did a meta-review or received a meta-review'
+            performed_at = Activity.performed_at
+            updated_at = Activity.updated_at
+            Participant.add_activity(name,
```

Comment:
Seems this is wrong. You should find the corresponding `gamification participant`. And do sth like `gamification_participant.add_activity()` instead of `Participant.add_activity`.